setup/src/main/java/io/jenkins/jenkinsfile/runner/JenkinsfileRunnerRule.java

```diff
@@ -47,13 +42,8 @@ public JenkinsfileRunnerRule(File warDir, File pluginsDir) {
      */
     @Override
     protected ServletContext createWebServer() throws Exception {
-        server = new Server(new ThreadPoolImpl(new ThreadPoolExecutor(10, 10, 10L, TimeUnit.SECONDS, new LinkedBlockingQueue<Runnable>(),new ThreadFactory() {
-            public Thread newThread(Runnable r) {
-                Thread t = new Thread(r);
-                t.setName("Jetty Thread Pool");
-                return t;
-            }
-        })));
+        QueuedThreadPool queuedThreadPool = new QueuedThreadPool(10);
```

Comment:
Removed custom Thread Pool like @olamy did for Winstone in https://github.com/jenkinsci/winstone/pull/54 . Likely we need to upstream the fix to Jenkins Test Harness so that it runs reliably on 50+ core machines. I wish we had them on our infra :P 

FYI @jglick